### PR TITLE
client: de-lint

### PR DESCRIPTION
**Pull request by strib**
Issue: HOTPOT-638

### CODE REVIEW

File: go/client/cmd_ctl.go [hunk 1]
```diff
@@ -36,15 +34,15 @@ func NewCmdCtl(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
 }
 
 // availableComponents specify which components can be included or excluded
-var availableCtlComponents = []string{
+var availableCtlComponents = []string{ //nolint
```
Comment (joshblum) [inline go/client/cmd_ctl.go]: just curious: why the `nolint` for these?

File: go/client/rpc.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ func getSocket(g *libkb.GlobalContext, clearError bool) (xp rpc.Transporter, err
 	var isNew bool
 	_, xp, isNew, err = g.GetSocket(clearError)
 	if err == nil && isNew {
-		introduceMyself(g, xp)
+		_ = introduceMyself(g, xp)
```
Comment (joshblum) [inline go/client/rpc.go]: why not return this error?

File: go/client/ui.go [hunk 3]
```diff
@@ -770,34 +770,34 @@ func (l LoginUI) DisplayPaperKeyPhrase(_ context.Context, arg keybase1.DisplayPa
...
-	l.parent.Printf("Your paper key is\n\n")
-	l.parent.Printf("\t")
-	l.parent.OutputDesc(OutputDescriptorPrimaryPaperKey, arg.Phrase)
-	l.parent.Printf("\n\n")
-	l.parent.Printf("Write it down....now!\n\n")
+	_, _ = l.parent.Printf("\n")
+	_, _ = l.parent.Printf("===============================\n")
+	_, _ = l.parent.Printf("IMPORTANT: PAPER KEY GENERATION\n")
+	_, _ = l.parent.Printf("===============================\n\n")
+	_, _ = l.parent.Printf("During Keybase's alpha, everyone gets a paper key. This is a private key.\n")
```
Comment (joshblum) [inline go/client/ui.go]:
```suggestion
	_, _ = l.parent.Printf("This is a private key.\n")
```

File: go/client/cmd_ctl.go (hunk 1 above)
Comment (strib) [reply go/client/cmd_ctl.go]: They are only used when the build tag is `darwin`, and so seem unused when linting on Linux.  I guess we could move them into a darwin-only file but this was faster to do at the time.

File: go/client/rpc.go (hunk 2 above)
Comment (strib) [reply go/client/rpc.go]: Good question, I don't remember having a reason.  I'll change it.

**Comment by strib** [comment]
Oops I merged this during code freeze by accident.  I've reverted it and will open a new PR.

**Pull request merged**